ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -289,6 +289,43 @@ By default an MLAG ID will only be configured on Port-Channel downlinks dual-hom
 + mlag_on_orphan_port_channel_downlink: true
 ```
 
+### Changes in MTU values and support for `p2p_uplinks_mtu` in platform settings
+
+#### Default MTU value changed from `9214` to `9194` for Dps1 and WAN HA interfaces
+
+In AVD 4.0.0, the default MTU for Dps1, LAN Uplink interfaces, and WAN HA interfaces was `9214`.
+In AVD 5.0.0, the default MTU for Dps1 and WAN HA interfaces has been changed to `9194`.
+
+#### New Feature: `p2p_uplinks_mtu` support in platform settings
+
+In AVD 5.0.0, support for `p2p_uplinks_mtu` has been added in platform settings. The precedence for setting MTU on point-to-point uplink interfaces is now as follows:
+
+1. `node_type_uplink_mtu`
+2. `platform_p2p_uplinks_mtu`
+3. `p2p_uplinks_mtu`
+4. Default value: `9214`
+
+MTU on WAN HA interfaces
+
+```diff
+wan_router:
+  node_groups:
+    - group: SITE_HA_ENABLED
+      wan_ha:
+        enabled: true
+        ha_interfaces: [Ethernet52]
++       mtu: 9214
```

Comment:
It is not clear that this is showing how to revert the new defaults.

Reply:
Here by default mtu set as `9194` if not defined so to retain the old mtu we have to use mtu: 9214 under wan_ha.

Reply:
I know, but you need to put some text above to explain that to the user :)

Reply:
added for WAN HA interfaces, Shell i add for Dps1 interface to use custom_structured_configuration_dps_interfaces to retain the old MTU?

Reply:
> added for WAN HA interfaces, Shell i add for Dps1 interface to use custom_structured_configuration_dps_interfaces to retain the old MTU?

Yeah for now I think that is fine.

Reply:
added